h2o-core/src/main/java/water/api/schemas3/RapidsHelpV3.java

```diff
@@ -4,27 +4,22 @@
 import water.api.API;
 
 /**
+ * Help for the rapids language
  */
 public class RapidsHelpV3 extends SchemaV3<Iced, RapidsHelpV3> {
 
-  @API(help="Rapids language, organized in a form of a tree (so that similar constructs are grouped together.",
+  @API(help="Description of the rapids language.",
       direction=API.Direction.OUTPUT)
-  public RapidsExpressionV3 syntax;
+  public RapidsExpressionV3[] expressions;
 
   public static class RapidsExpressionV3 extends SchemaV3<Iced, RapidsExpressionV3> {
     @API(help="(Class) name of the language construct")
     public String name;
 
-    @API(help="If true, then this is not a standalone construct but purely a grouping level.")
-    public boolean is_abstract;
```

Comment:
That was initial idea - however I decided to keep it clean since the call was crashing anyway before. If it would be working, I would keep it there, but right now I think it's safe to delete it. We can add it back in the future if needed.